src/components/RightSidebar/RightSidebar.vue

```diff
@@ -4,14 +4,27 @@
 -->
 
 <template>
-	<NcAppSidebar v-show="opened"
+	<NcAppSidebar v-if="token"
+		:open="opened"
 		:name="name"
 		:title="name"
 		:active="activeTab"
 		:class="'active-tab-' + activeTab"
+		:toggle-classes="{ 'chat-button': isInCall }"
+		:toggle-attrs="isInCall ? inCallToggleAttrs : undefined"
+		@update:open="handleUpdateOpen"
 		@update:active="handleUpdateActive"
 		@closed="handleClosed"
 		@close="handleClose">
+		<!-- Use a custom icon when sidebar is used for chat messages during the call -->
+		<template v-if="isInCall" #toggle-icon>
+			<MessageText :size="20" />
+			<NcCounterBubble v-if="unreadMessagesCounter > 0"
+				class="chat-button__unread-messages-counter"
```

Comment:
![image](https://github.com/nextcloud/spreed/assets/93392545/5b542008-7ca6-4c77-b566-ff1214f309f2)

we don't need to bury class under :deep selector, I think

Reply:
For `chat-button__unread-messages-counter` deep is not needed, but for the `chat-button` itself it is needed.

Let me check with 29 styles...

Reply:
Working on a regression

Reply:
Thanks for noticing the problem